reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -324,7 +324,7 @@ class StaticHtmlReporter : Reporter {
         val cssClass = if (areAllResolved) {
             "ort-resolved"
         } else {
-            when (issues.map { it.severity }.min() ?: Severity.ERROR) {
+            when (issues.filter { it.isResolved }.map { it.severity }.min()!!) {
```

Comment:
Commit message nit: "resovled" -> "resolved"

Nit: Changing `?: Severity.ERROR` to `!!` would make more sense in the previous commit, because you moved the expression into the where branch there.

Reply:
Also commit message: "Ingore" -> "Ignore "